person.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'nameable'
+
```

Comment:
### SUGGESTION 

- [ ] To run your app I had to add the `require_relative` script to import the `capitalize_decorator` and `trimmer_decorator`, also I removed the `@parent_permission = parent_permission` after doing this the app works successfully 👍🏿 

![Screenshot_130](https://github.com/AbdelazizSoliman/OOP-school-library/assets/69638013/ecddbbe8-760e-4283-848a-3d7b8e6f558e)